README.md

```diff
@@ -0,0 +1,5 @@
+# ng2-jspm-template-lib
+
+[![Build Status](https://travis-ci.org/flaviait/ng2-jspm-template-libs.svg?branch=master)](https://travis-ci.org/flaviait/ng2-jspm-template-libs)
+
+This is a tooling library for the [ng2-jspm-template](https://github.com/flaviait/ng2-jspm-template).
```

Comment:
This description should be a little precise:
- Which tools are included?
- How can they be used?
- Is it possible to use all or some of them without the template? Imho some of the tasks might be interesting for other projects as well.